daemon/protocol/python/tests/daemon_cli_test.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python
+
+import os
+import re
+import shutil
+import subprocess
+import tempfile
+import time
+import unittest
+from Carbon.Aliases import false
+
+gopath = os.environ['GOPATH']
+pypath = gopath + '/src/github.com/scootdev/scoot/daemon/protocol/python/scoot'
+cliRef = ['python', pypath+'/daemon_cli.py']
+createS = ['snapshot', 'create']
+checkoutS = ['snapshot', 'checkout']
+runE = ['exec', 'run']
+pollE = ['exec', 'poll']
+
+echo = ['echo']
+
+class TestCliCommands(unittest.TestCase):
+    daemonProcess = None
+    pypath = ''
+      
+    def setUp(self):
+        self.tmpdir = tempfile.mkdtemp()
+
+        # Note: the following does not work - the process from the pool does not have GOPATH defined so it can't find the binary
+        try:
+            self.daemonProcess = subprocess.Popen(['{0}/bin/daemon'.format(gopath), '-execer_type', 'os'])
+        except Exception as e:
```

Comment:
What specific exception are we trying to catch here? Catch-alls are unpythonic, if we should make it specific like OSError, or ditch the try block altogether.